great-idea-website/css/index.css

```diff
@@ -130,13 +156,30 @@ header img {
 	font-size: 16px;
 }
 
+@keyframes button-animation{
+	0% {width: 50%;}
+	100% {width: 100%;}
```

Comment:
This animation again fits in the category of subtle and appropriate transitions and I really thought it was a great addition.  Good idea using the widths here...